linux_os/guide/system/software/gnome/package_gdm_removed/rule.yml

```diff
@@ -38,3 +38,6 @@ template:
     name: package_removed
     vars:
         pkgname: gdm
+        pkgname@ubuntu1604: gdm3
```

Comment:
There is also template in shared/templates/extra_ovals.yml which is about gdm.

Reply:
Good catch, thanks @vojtapolasek!